File: MPI/mpicheck/tests/functions/1.0/MPI_Cart_sub.f90

        
        program main
        use mpi
        
        INTEGER var_0
       LOGICAL var_1(10)
       INTEGER var_2
       INTEGER var_3
        call mpi_cart_sub(var_0, var_1, var_2, var_3)
       call pmpi_cart_sub(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Cart_sub.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    const int var_1[2];
    MPI_Comm *var_2;
    int ret;
    /* calls */
    ret = MPI_Cart_sub(var_0, var_1, var_2);
    ret = PMPI_Cart_sub(var_0, var_1, var_2);
    return 0;
}
